files/ja/web/svg/element/svg/index.md

```diff
@@ -9,6 +9,8 @@ slug: Web/SVG/Element/svg
 
 > **メモ:** `xmlns` 属性は *SVG ドキュメント*の最も外側の `svg` 要素にのみ必要です。内部の `svg` 要素や HTML 文書の内部には不要です。
 
+<h2 id="Example">例</h2>
```

Comment:
Markdownの構文を使用してください。
現在は、EmbedLiveSampleと見出しのIDが一致していなくても、文書構造から表示するサンプルを割り出すようになっています。
```suggestion
## 例
```

Reply:
Just in case for who wants know what we disscussed in this review, I'll wrote English translated review below.
@mfuji09 said: Use Markdown syntax. As of now, the live sample view is assumed/detected from document stracture whatever `EmbedLiveSample`'s ID and Heading ID isn't matched.

Reply:
@mfuji09 are you sure that will be work? Because I copied `<h2 id="Example">例</h2>` from another page – https://github.com/mdn/translated-content/blob/main/files/ja/web/svg/element/stop/index.md?plain=1#L10

Reply:
If you're sure – 👍

Reply:
@sashasushko 
It was translated when the `EmbedLiveSample` macro didn't work correctly in such a case. Today such a correction shouldn't be used because the macro seems to work correctly now.